bundles/extensions/healthcheck/core/src/main/java/org/apache/sling/hc/api/HealthCheck.java

```diff
@@ -66,7 +66,13 @@
      * will be executed asynchronously using the cron expression provided.
      */
     String ASYNC_CRON_EXPRESSION = "hc.async.cronExpression";
-    
+
+    /**
+     * Optional service property: Time To Leave for health check {@link Result}.
```

Comment:
This property should also be made available via annotation https://github.com/drfits/sling/blob/trunk/bundles/extensions/healthcheck/annotations/src/main/java/org/apache/sling/hc/annotations/SlingHealthCheck.java